src/platform/forms-system/src/js/review/ArrayField.jsx

```diff
@@ -207,7 +210,9 @@ class ArrayField extends React.Component {
       <div className={itemsNeeded ? 'schemaform-review-array-warning' : null}>
         {title && (
           <div className="form-review-panel-page-header-row">
-            <h3 className="form-review-panel-page-header">{title}</h3>
+            <h3 className="form-review-panel-page-header vads-u-font-size--h4">
```

Comment:
This added class displays an `h3` as an `h4`. Has this passed design review?

cc @CrystabelReiter